specification/compute/resource-manager/Microsoft.Compute/DiskRP/stable/2022-07-02/snapshot.json

```diff
@@ -561,6 +561,10 @@
           "type": "boolean",
           "description": "Whether a snapshot is incremental. Incremental snapshots on the same disk occupy less space than full snapshots and can be diffed."
         },
+        "incrementalSnapshotFamilyId": {
+          "type": "string",
+          "description": "The id of the family of incremental snapshot for the same disk."
```

Comment:
Add: Incremental snapshots for a disk share an incremental snapshot family id. The Get Page Range Diff API can only be called on incremental snapshots with the same family id.

Reply:
Done